Latest/Interface/Base.lproj/Main.storyboard

```diff
@@ -450,11 +450,28 @@
                                         <autoresizingMask key="autoresizingMask" flexibleMaxX="YES" flexibleMinY="YES"/>
                                     </progressIndicator>
                                 </toolbarItem>
+                                <toolbarItem implicitItemIdentifier="5B354B41-598E-4279-9FD9-F493AEA477C5" label="Update All" paletteLabel="Update All" image="NSTouchBarDownloadTemplate" id="qS5-vO-h2j">
```

Comment:
Thanks for adding that back in! I previously had the icon to the left of the reload button. Not sure if that would look better or worse now. 

What OS are you on? Looks great on Big Sur. If you are on Catalina or lower, I would be happy to see a screenshot. If not, no worries, I can boot up my trusty VM ;)